src/scripts.js

```diff
@@ -127,12 +284,48 @@ function makeRandomDate(userStorage, id, dataSet) {
 }
 
 function addHydrationInfo(id, hydrationInfo, dateString, userStorage, laterDateString) {
-  hydrationToday.insertAdjacentHTML('afterBegin', `<p>You drank</p><p><span class="number">${hydrationInfo.calculateDailyOunces(id, dateString)}</span></p><p>oz water today.</p>`);
+  // hydrationToday.innerHTML = '';
```

Comment:
As with all the comments (and console.logs) in scripts right now, I do have those all cleaned up for the next PR!